doc/user_guide.md

```diff
@@ -30,20 +30,41 @@ Please refer to spdm_client_init() in [spdm_requester.c](https://github.com/DMTF
 
 1. Initialize SPDM context
 
-   1.1, allocate buffer for the spdm_context and initialize it.
+   1.1, allocate buffer for the spdm_context, initialize it, and setup scratch_buffer.
+   The spdm_context may include the decrypted secured message or session key.
+   The scratch buffer may include the decrypted secured message.
+   The spdm_context and scratch buffer shall be zeroed before freed or reused.
 
    ```
    spdm_context = (void *)malloc (libspdm_get_context_size());
    libspdm_init_context (spdm_context);
+
+   scratch_buffer_size = libspdm_get_sizeof_required_scratch_buffer(m_spdm_context);

```

Comment:
Integrator should also be able to get the size statically to be used in a linker script.

Reply:
Are you proposing to remove get_scratch_buffer_size, but define a MACRO in spdm_common_lib.h ?

Reply:
Both ways should be available. It can be called `LIBSPDM_SCRATCH_BUFFER_SIZE` or something similar.

Reply:
Doc updated